### DISCUSSION TITLE
Testing one, two, three

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1128] [Team Posts]
G'day.

**Comment by qknight** [comment]
Hi

On December 6, 2019 6:52:50 PM GMT+01:00, Tim <notifications@github.com> wrote:
>G'day.
>
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/orgs/muse-sequencer/teams/musedevteam/discussions/1

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by kybos** [comment]
Hello.

**Comment by spamatica** [comment]
What do you know, there is a dev mailinglist :)



Den fre 6 dec. 2019 kl 19:14 skrev kybos <notifications@github.com>:

**Comment by terminator356** [comment]
Testing reply directly from email client...

"How 'bout that local sports team, eh?"

  - Monty Burns awkwardly sitting down to dinner
     with the Simpsons.
